test/patterns/method_chaining/EmptyMethodChain.java

```diff
@@ -0,0 +1,35 @@
+class Test {
+    public void start() {
+        final JndiService jndiService = serviceRegistry
+                .getService(JndiService.class);
+        final ConnectionFactory jmsConnectionFactory = jndiService
+                .locate(jmsConnectionFactoryName);
+
+        this.jmsSession = jmsConnection.createSession(
+                true,
+                Session.AUTO_ACKNOWLEDGE
+        );
+
+		System.out.println("asdasd" + aaa + "34234" + bbb);
+        list = new ArrayList<>();
+        for (int i = 0; i < 10; i++)
+            list.add(Boolean.FALSE);
+        list = new ArrayList<>();
+        for (int i = 0; i < 10; i++)
+            list.add(Boolean.FALSE);
+
+        CriteriaCollection().instance()
```

Comment:
@lyriccoder Please correct this indentation. I know that this isn't a production class but it will be better with the correct indent.